src/Shipping/ShippingController.php

```diff
@@ -60,6 +61,25 @@ function() {
 		add_filter( 'pre_update_option_pickup_location_pickup_locations', array( $this, 'flush_cache' ) );
 	}
 
+	/**
+	 * Gets a list of payment method ids that support the 'collectible' feature.
+	 *
+	 * @return string[] List of payment method ids that support the 'collectible' feature.
+	 */
+	public function get_collectible_method_ids() {
+		return array_keys(
+			wp_list_pluck(
+				array_filter(
+					WC()->shipping()->get_shipping_methods(),
+					function( $method ) {
+						return in_array( 'collectible', $method->supports, true );
```

Comment:
Did we avoid [this method](https://github.com/woocommerce/woocommerce/blob/3611d4643791bad87a0d3e6e73e031bb80447417/plugins/woocommerce/includes/abstracts/abstract-wc-shipping-method.php#L154-L156) for a reason?

Reply:
I'm using it now, thanks for showing me it.